lectures/python_essentials.md

```diff
@@ -940,14 +933,27 @@ Here's a solution:
 
 ```{code-cell} python3
```

Comment:
To show my changes to the exercise solution clearly, here is the solution that I have rewritten.